addons/binding/org.openhab.binding.netatmo/README.md

```diff
@@ -92,6 +92,10 @@ Number Netatmo_Indoor_CO2 "CO2" <carbondioxide> { channel = "netatmo:NAMain:home
 * Location
 * TimeStamp
 * LastStatusStore
+* MinTemp
+* MaxTemp
```

Comment:
I found a volunteer. Now we need a .jar file with support for the new channels on the NAModule4. I can build it on my own, but I don't know if we must change something in the underlying swagger library as well.